#,Module,TestFunction,Result,TimeTaken,ExecutedOn,Syslogs,FCLI,TSSH,DCNT,Description,Devices,KnownIssue,Doc
1,switching/test_portchannel.py,,,0:00:48,2023-12-29 02:48:32,0,1,0,2,Module Prolog,"V3, V4",,
2,switching/test_portchannel.py,test_ft_portchannel_with_vlan_variations,Fail,0:01:52,2023-12-29 02:50:15,0,1,0,2,"Portchannel PortChannel7 on V4 with member ['Ethernet48', 'Ethernet52'] verification failed","V3, V4",,"This test case covers below test scenarios/tests FtOpSoSwLagFn041 : Verify that port-channel is up or not when port-channel created followed by add it to VLAN and then making the port-channel up. FtOpSoSwLagFn042 : Verify that port-channel is up when port-channel is created, making the port-channel up and then adding the port-channel to VLAN"
3,switching/test_portchannel.py,test_ft_verify_min_links_functionality_by_flapping_all_member_ports_007,ConfigFail,0:00:40,2023-12-29 02:50:54,0,1,0,2,pre-test configuration failed Invalid error code PortChannel Verification Failed : unknown message identifier 'PortChannel Verification Failed' @5430,"V3, V4",,
4,switching/test_portchannel.py,,,0:00:32,2023-12-29 02:51:26,0,1,0,2,Module Epilog,"V3, V4",,
5,system/test_ssh.py,,,0:01:09,2023-12-29 02:54:41,0,1,0,2,Module Prolog,"V3, V4",,
6,system/test_ssh.py,test_ft_ssh_service_disable,Pass,0:00:08,2023-12-29 02:54:44,0,1,0,2,Test case passed,"V3, V4",,
7,system/test_ssh.py,test_ft_ssh_add_user_verify,Fail,0:09:14,2023-12-29 03:03:58,0,1,0,2,Test case Failed.,"V3, V4",,
8,system/test_ssh.py,test_ft_verify_logs_at_the_time_of_ssh_login_success,Pass,0:00:21,2023-12-29 03:04:18,0,1,0,2,Test case passed,"V3, V4",,"ceta-sonic-67575: Verify that on successful ssh login, error logs are generated"
9,system/test_ssh.py,,,0:00:51,2023-12-29 03:05:09,0,1,0,2,Module Epilog,"V3, V4",,
